packages/cache-inmemory/src/index.ts

```diff
@@ -0,0 +1,13 @@
+import { ModuleExports } from "@medusajs/medusa"
+
+import InMemoryCacheService from "./services/inmemory-cache"
+
+const loaders = []
+const service = InMemoryCacheService
+
+const moduleDefinition: ModuleExports = {
+  service,
```

Comment:
FYI this [PR](https://github.com/medusajs/medusa/pull/3329/files#diff-c7b62e8108f9340c97e8576ca2692105f91742dd659d457f903ce02cb036ca25R89) will change that export to accept multiple services. Also the type ModuleExports is located at "@medusajs/module-sdk".
No action needed for now. If this is merged before I can apply the changes.

Reply:
Will remove that @adrien2p , indeed only services are required to be exported